components/src/dynamo/planner/monitoring/dgd_services.py

```
@@ -101,46 +137,45 @@ def get_component_name_from_endpoint_arg(self) -> Optional[str]:
+                f"Please set main container resources.limits.{GPU_RESOURCE_KEY} "
+                f"or resources.requests.{GPU_RESOURCE_KEY} in the DGD."
             )
 
         try:
             return int(gpu_str)
         except (ValueError, TypeError):
             raise ValueError(
-                f"Invalid GPU count '{gpu_str}' for service '{self.name}'. "
+                f"Invalid GPU count '{gpu_str}' for component '{self.name}'. "
                 f"GPU count must be an integer."
             )
```

Ruff (0.15.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)